src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/NewBlogPostStoringActor.java

```diff
@@ -1,46 +1,57 @@
 package pl.tomaszdziurko.jvm_bloggers.blog_posts;
 
+import static com.google.common.base.MoreObjects.firstNonNull;
 
 import akka.actor.AbstractActor;
 import akka.actor.Props;
 import akka.japi.pf.ReceiveBuilder;
+
+import com.sun.syndication.feed.synd.SyndContent;
 import com.sun.syndication.feed.synd.SyndEntry;
+
+import lombok.experimental.ExtensionMethod;
 import lombok.extern.slf4j.Slf4j;
 import pl.tomaszdziurko.jvm_bloggers.blog_posts.domain.BlogPost;
 import pl.tomaszdziurko.jvm_bloggers.blog_posts.domain.BlogPostRepository;
 import pl.tomaszdziurko.jvm_bloggers.utils.DateTimeUtilities;
 
-import java.time.LocalDateTime;
 import java.util.Date;
-import java.util.Optional;
+
+import org.apache.commons.lang3.StringUtils;
 
 @Slf4j
+@ExtensionMethod({DateTimeUtilities.class, StringUtils.class})
 public class NewBlogPostStoringActor extends AbstractActor {
 
-    private final BlogPostRepository blogPostRepository;
-
     public NewBlogPostStoringActor(BlogPostRepository blogPostRepository) {
```

Comment:
By code-free I meant that only object initialziation can take place in constructor whereas businnes logic should be avoided. I didn't noticed that this is object building, not code execution. Of course your code i totaly fine :)